WordCountScrapper/Main.java

```diff
@@ -0,0 +1,142 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.*;
```

Comment:
Import all required packages explicitly, * import is a bad practice

Reply:
Removed it.